codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/smoketests/SmokeTestsRunnerGenerator.kt

```diff
@@ -0,0 +1,174 @@
+package software.amazon.smithy.kotlin.codegen.rendering.smoketests
+
+import software.amazon.smithy.codegen.core.Symbol
+import software.amazon.smithy.codegen.core.SymbolProvider
+import software.amazon.smithy.kotlin.codegen.core.*
+import software.amazon.smithy.kotlin.codegen.model.getTrait
+import software.amazon.smithy.kotlin.codegen.rendering.util.render
+import software.amazon.smithy.kotlin.codegen.utils.dq
+import software.amazon.smithy.kotlin.codegen.utils.getOrNull
+import software.amazon.smithy.kotlin.codegen.utils.toCamelCase
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.smoketests.traits.SmokeTestCase
+import software.amazon.smithy.smoketests.traits.SmokeTestsTrait
+import kotlin.jvm.optionals.getOrNull
+
+/**
+ * Renders smoke tests runner for a service
+ */
+class SmokeTestsRunnerGenerator(
+    private val writer: KotlinWriter,
+    private val service: Symbol,
+    private val operations: List<OperationShape>,
+    private val model: Model,
+    private val symbolProvider: SymbolProvider,
+    private val sdkId: String,
+) {
+    internal fun render() {
+        writer.write("import kotlin.system.exitProcess")
```

Comment:
You should not need to import anything yourself, using the `#T` symbol should add the imports automatically

Reply:
I had some issues here doing that. The `exitProcess` function is not available in common, only JVM and Native. See: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/exit-process.html.